app/controllers/provider_interface/provider_users_controller.rb

```diff
@@ -82,6 +82,21 @@ def update_providers
       redirect_to provider_interface_provider_user_path(provider_user)
     end
 
+    def confirm_remove
+      @provider_user = ProviderUser.visible_to(current_provider_user).find(params[:provider_user_id])
+    end
+
+    def remove
+      @provider_user = ProviderUser.visible_to(current_provider_user).find(params[:provider_user_id])
```

Comment:
This can use `find_provider_user` from https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2013 now.

Reply:
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2012/commits/481fdb517c6bb01a4f1c34e85eee421dc316760b#diff-6c36ce09043ee29972a80d5a8862669dR78-R82 :+1: